parsl/executors/high_throughput/interchange.py

```diff
@@ -549,7 +549,6 @@ def process_results_incoming(self, interesting_managers: Set[bytes], monitoring_
                         monitoring_radio.send(r['payload'])
                     elif r['type'] == 'heartbeat':
                         logger.debug("Manager %r sent heartbeat via results connection", manager_id)
-                        b_messages.append((p_message, r))
                     else:
```

Comment:
the message is an awkwardly phrased message that the manager has sent a heartbeat to the interchange *and we have received it in the interchange* - what's actually happened is that we have received it, not that it was sent.

the line remove stops a further sending happen.

I have a patch to rephrase that message and another one like it for the other heartbeat...

I was saving removing the whole of this result-side heartbeat path (as discussed in #3464) until later - because I'm slightly more suspicious about if this is doing meaningful TCP-level keepalive despite not being handled at the parsl protocol layer.